src/components/CountryDetails/CountryDetails.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { fetchCountries } from '../../redux/worldCountries/worldCountries';
+import Cards from './Cards';
+import styles from './CountryDetails.module.css';
+
+const CountryDetails = () => {
```

Comment:
### Ensure that these functionalities reflect in your live demo as well
<hr />

- [ ] You app is missing the header as contained in the design. Kindly add it.

  ![ixboy-navbar](https://user-images.githubusercontent.com/36542986/158457125-b8528ad3-f6ae-4363-b99d-56ba74b77b02.jpg)
 
- [ ] When a user navigates to the **details** page on the top of the page, on the header above, there should be a `<` (**back**) button that navigates back to the Home page when clicks. Kindly add this functionality

  ![ixboy-back](https://user-images.githubusercontent.com/36542986/158457443-f262ab3d-078d-4ce4-bc58-81b0efa69e0e.jpg)